src/modules/flight_mode_manager/tasks/AutoFollowTarget/FlightTaskAutoFollowTarget.hpp

```diff
@@ -37,63 +37,94 @@
+
 // [m] max vertical deviation from position setpoint, above
 // which no horizontal control is done
 static constexpr float ALT_ACCEPTANCE_THRESHOLD = 3.0f;
 
-// Vertical ascent speed when the drone detects that it
-// is too close to the ground (below MINIMUM_SAFETY_ALTITUDE)
-static constexpr float EMERGENCY_ASCENT_SPEED = 0.2f;
+// [m] Minimum distance between drone and target for the drone to do any yaw control.
+static constexpr float MINIMUM_DISTANCE_TO_TARGET_FOR_YAW_CONTROL = 1.0f;
+
+// << Target Position Velocity Estimator related constants >>
+
+// [rad/s] Second Order reference model filter natural frequency
+static constexpr float TARGET_POS_VEL_FILTER_NATURAL_FREQUENCY = 1.0f;
+
+// [.] Second Order reference model filter damping ratio
+static constexpr float TARGET_POS_VEL_FILTER_DAMPING_RATIO = 0.7071;
+
+// [us] If the target estimator output isn't updated longer than this, reset pose filter.
+static constexpr uint64_t TARGET_ESTIMATOR_TIMEOUT_US = 1500000UL;
+
+// << Target Course Angle Tracking related constants >>
 
-// Filter on setpoints for smooth cinematic experience:
-// Lowpass applied to the estimated position of the target
-// before using it as control input
-static constexpr float POSITION_FILTER_ALPHA =	1.5f;
+// [m/s] Speed deadzone for which, under this velocity, the target orientation
+// tracking will freeze, since orientation can be noisy in low velocities
+static constexpr float TARGET_SPEED_DEADZONE_FOR_ORIENTATION_TRACKING = 1.0;
 
-// Filter on setpoints for smooth cinematic experience:
-// Lowpass applied to the follow-me angle setting, to ensure
-// smooth and circular transitions between settings
-static constexpr float FOLLOW_ANGLE_FILTER_ALPHA = 3.0f;
+// << Orbit Angle control related constants >>
 
-// Filter on setpoints for smooth cinematic experience:
-// Lowpass applied to the actual NED direction how the drone is facing the target
-// regarless of the setting. Used for dynamic tracking angles when the target makes a turn
-static constexpr float DIRECTION_FILTER_ALPHA =	3.0f;
+// [m/s^3] Maximum Jerk setting for generating the Follow Target Orbit trajectory
+static constexpr float ORBIT_TRAJECTORY_MAX_JERK = 4.0;
 
-// Filter on setpoints for smooth cinematic experience:
-// Lowpass applied for ramping up / down velocity feedforward term.
-// This is to avoid aggressive jerks when the target starts moving, because
-// velocity feed-forward is not applied at all while the target is stationary.
-static constexpr float VELOCITY_FF_FILTER_ALPHA = 1.0f;
+// [m/s^2] Maximum acceleration setting for generating the Follow Target Orbit trajectory
+static constexpr float ORBIT_TRAJECTORY_MAX_ACCELERATION = 2.0;
+
+// << RC Adjustment related constants >>
+
+// [m/s] Speed with which the follow distance will be adjusted by when commanded with deflection via RC command
+static constexpr float FOLLOW_DISTANCE_USER_ADJUST_SPEED = 2.0;
+// [m] Maximum follow distance that can be set by user's RC adjustment
```

Comment:
I see the sentiment here where you wanted to group similar entries together by omitting a new line in between them. Personally though I think it lowers the readability, because it's an irregularity.

Reply:
Sure! :)